src/sage/schemes/hyperelliptic_curves/monsky_washnitzer.py

```diff
@@ -717,25 +722,25 @@ def helper_matrix(Q):
 
 def lift(x):
     r"""
-    Try to call x.lift(), presumably from the `p`-adics to ZZ.
+    Try to call ``x.lift()``, presumably from the `p`-adics to ``ZZ``.
```

Comment:
In other places you used `\ZZ`, but both are ok I think.

Reply:
I've changed it, as part of 1b873b9ec91